src/framework/standard/mod.rs

```diff
@@ -383,6 +383,35 @@ impl StandardFramework {
         self
     }
 
+    /// Adds a group to be used by the framework. Primary use-case is runtime modification
+    /// of groups in the framework; will _not_ mark the framework as initialized. Refer to
+    /// [`group`] for adding groups in initial configuration.
+    /// 
+    /// Note: does _not_ return `Self` like many other commands. This is because
+    /// it's not intended to be chained as the other commands are.
+    /// 
+    /// [`group`]: #method.group
+    pub fn group_add(&mut self, group: &'static CommandGroup) {
+        // Implementation is duplicated from above group function
```

Comment:
Then could you alter `group` to use this method? I'd rather not want to be shooting my, or anyone else's foot when committing modifications to `group` and forgetting to apply those to `group_add` as well or vice versa.

Reply:
Changed!

Sorry for the delay, apparently I totally missed that you commented this...